ddl/backfilling_scheduler.go

```diff
@@ -339,7 +339,7 @@ func (b *ingestBackfillScheduler) close(force bool) {
 		}
 	}
 	close(b.resultCh)
-	if intest.InTest && len(b.copReqSenderPool.srcChkPool) != copReadChunkPoolSize() {
```

Comment:
There should be other places in TiDB to use this, why only change here?

Reply:
Just test it. I don't know why @tiancaiamao's changement will fail in the CI>